package/com.unity.formats.usd/Tests/Runtime/SanitizedSamplesTests.cs

```diff
@@ -235,13 +233,12 @@ public void UniformToFaceVarying_ValuesAreCorrect()
     public class SanitizedInstancesTests
     {
         GameObject m_usdRoot;
-        string m_usdGUID = "c5046ed8700c010469c4c557353bc241"; // GUI of mesh_instances.usda
 
         [SetUp]
         public void SetUp()
         {
             InitUsd.Initialize();
-            var usdPath = Path.GetFullPath(AssetDatabase.GUIDToAssetPath(m_usdGUID));
+            var usdPath = Path.GetFullPath(AssetDatabase.GUIDToAssetPath(TestAssetData.GUID.Instancer.instancedUsd));
```

Comment:
I think this should have 'mesh' in the name, as 'instanced' is too vague. Alternatively keep the definition in this file as it's not used anywhere else so doesn't need to be abstracted :)

Reply:
Changed variable name to MeshInstancesUsd